src/Solnet.Rpc/Builders/MessageBuilder.cs

```diff
@@ -96,12 +61,17 @@ internal int Length()
         /// <summary>
         /// The list of instructions contained within this transaction.
         /// </summary>
-        internal readonly List<TransactionInstruction> Instructions;
+        internal List<TransactionInstruction> Instructions { get; private set; }

```

Comment:
its already internal, so I think the private set should either apply to all members, or none.
As long as the API consumer can't access this and screw up, should be good, but need consistency tho'